tests/suites/main_test.function

```diff
@@ -140,14 +140,19 @@ int get_line( FILE *f, char *buf, size_t len )
 {
```

Comment:
Since we are filtering some lines here. Perhaps this function should be called ```get_test_line```.
Alternatively filtering can also be done at https://github.com/ARMmbed/mbedtls/pull/1116/files#diff-1c3934ecaaf7da8b368603ce9f12de77L359

Reply:
@mazimkhan I prefered not to change anything in `main` so that the change would be minimal, both in terms of semantics (ignore any line beginning with `#`) and in terms of implementation (localized to one function).